app/services/create_reservation.rb

```diff
@@ -1,32 +1,39 @@
 # frozen_string_literal: true
 
 class CreateReservation
+
+  def initialize(user_id, screening_id, seats)
+    @user_id = user_id
+    @screening_id = screening_id
+    @seats = seats
+  end
+
   def call
     return false unless seats_selected?
```

Comment:
Ideally, it would be to return an object which can store validation errors in order to present them and allow the user to try again.

You can do it by returning an object which implements `#errors` & `#full_messages` (https://github.com/danyflorczak/Monte_Cinema/blob/refactor/app/views/shared/_errors.html.erb#L6)

Notice that you create here a "Reservation" (business entity) which is `Reservation` + `tickets` on the database layer (implementation).